docs/docs/sample-scripts/aad/replace-membership-of-selected-groups.md

```diff
@@ -1,3 +1,9 @@
+---
+tags:  
+  - Cleanup
```

Comment:
Since we tagged `Replace an owner in a Microsoft 365 Group or Microsoft Team` with `security,users`, we should use the same tags here as well